yash-datalake/azuredeploy.json

```diff
@@ -147,6 +147,13 @@
       },
       "defaultValue": "[toLower(uniqueString(resourceGroup().id, 'datafactorya'))]"
     },
+    "appName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Azure data lake UI app registered. Must be globally unique."
+      },
+      "defaultValue": "[toLower(uniqueString(resourceGroup().id, 'azuredatalakeuiappa'))]"

```

Comment:
the CI failure for this run was due to the eventhub name starting with a number... we should flip the uniqueString functions to start with the literal string and the unique string appended... e.g.

  "defaultValue": "[toLower(concat('azuredatalakeuiappa, uniqueString(resourceGroup().id)))]", 

You'll have to check for max length on the names for this as well...

Reply:
Yes done with required changes